src/android/setup.cfg

```diff
@@ -57,7 +57,4 @@ exclude=\
     build/*
 max-complexity = 10
 max-line-length = 119
```

Comment:
As with isort, it would be much easier to understand and maintain if there was just one flake8 configuration covering the whole repository.